packages/cli/src/GsnTestEnvironment.ts

```diff
@@ -75,7 +77,7 @@ class GsnTestEnvironmentClass {
       stake: ether('1'),
       funds: ether('1'),
       relayUrl: relayUrl,
-      gasPrice: '1e9',
+      gasPrice: 1e9.toString(),
```

Comment:
If I understand this change correctly, ganache didn't care about the inputs but hardhat does?

Reply:
No, ganahce read it as 0x1e9 === 489, and it was fine. Hardhat has baseFee higher than that due to london fork, so it needs more than 489 gasPrice.

Reply:
😂🤦